docs/tutorials/paroutes_benchmark.ipynb

```diff
@@ -0,0 +1,761 @@
+    "\n",
+    "and_node_cost_fn = ReactionModelLogProbCost(normalize=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "29e949b1-abc5-4514-aba7-7f86571425a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#3: search heuristic (value function)\n",
+    "# Here we just use a constant value function which is always 0,\n",
+    "# corresponding to the \"retro*-0\" algorithm (the most optimistic)\n",
+    "\n",
+    "from syntheseus.search.node_evaluation.common import ConstantNodeEvaluator\n",
+    "retro_star_value_function = ConstantNodeEvaluator(0.0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "2112a548-e2e1-4e85-873f-8654fac0b11f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "retro_star_alg = retro_star.RetroStarSearch(\n",
+    "    reaction_model=reaction_model,\n",
+    "    mol_inventory=inventory,\n",
+    "    or_node_cost_fn=or_node_cost_fn,\n",
+    "    and_node_cost_fn=and_node_cost_fn,\n",
+    "    value_function=retro_star_value_function,\n",
+    "    limit_reaction_model_calls=RXN_MODEL_CALL_LIMIT,\n",
+    "    time_limit_s=TIME_LIMIT_S,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "628e9336-3263-4dc3-991b-f62b2c1e6ac4",
+   "metadata": {},
+   "source": [
+    "### 2b: MCTS\n",
+    "\n",
+    "MCTS is an RL algorithm that tries to find routes of maximum value.\n",
+    "Standard MCTS works on an MDP where states are sets of molecules\n",
+    "and actions are backward reactions.\n",
+    "MCTS requires the following functions:\n",
+    "\n",
+    "1. A _reward function_ which defines the rewards for terminal states\n",
+    "2. A _value function_ which estimates the rewards attainable from a state\n",
+    "3. (optional, but very useful) a policy which assigns higher numbers to desirable actions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "af997a32-1f39-4788-b88a-463d92b7c9c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from syntheseus.search.graph.molset import MolSetNode, MolSetGraph\n",
+    "from syntheseus.search.node_evaluation.common import ReactionModelProbPolicy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "aa02eecf-71c9-4f17-bc21-5df514e7d4a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#1: reward function\n",
+    "# Here we use the most basic reward: 1.0 if a state is solved\n",
+    "# (all molecules are purchasable), and 0.0 otherwise.\n",
+    "from syntheseus.search.node_evaluation.common import HasSolutionValueFunction\n",
+    "\n",
+    "mcts_reward_function = HasSolutionValueFunction()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "5bfd9f29-58af-4dc5-b760-20ee65979777",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#2: value function\n",
+    "# while previous works have used rollouts,\n",
```

Comment:
nit: maybe start with uppercase ("While")

Reply:
Done (and fixed one other instance of this)